airqo-platform/AirQo-frontend
# Ft map filter enhancement

#### Summary of Changes (What does this PR do?)
This PR includes a change to the map filter icon and addition of UX elements to make it visible.

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
This branch should be run and changes will be see on the device management module

#### What are the relevant tickets?
- Github Issue: #505

#### Screenshots 
![Screenshot (37)](https://user-images.githubusercontent.com/53012890/156326762-f9a6c2f8-2551-4a35-b827-4a100ab1dda2.png)
